pkg/operator/hive/hive.go

```diff
@@ -156,6 +156,7 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 		"config/crds/hive_v1alpha1_selectorsyncset.yaml",
 		"config/crds/hive_v1alpha1_syncidentityprovider.yaml",
 		"config/crds/hive_v1alpha1_syncset.yaml",
+		"config/crds/hive_v1alpha1_syncsetinstance.yaml",
```

Comment:
We are missing more than just this yaml file. Should we instead apply all of the yaml files in the `config/crds` directory?